### PR TITLE
Fix "Already initialized constant" warnings when running

### LINKED ISSUE #296: Already initialized constant Rouge::Lexers:Shell::KEYWORDS
Now that GitLab is using Rouge, it appears that Rouge is loading files automatically, causing Rails to warn with errors:

```
/Users/stanhu/.rbenv/versions/2.1.6/lib/ruby/gems/2.1.0/gems/rouge-1.9.1/lib/rouge/lexers/shell.rb:20: warning: already initialized constant Rouge::Lexers::Shell::KEYWORDS
/Users/stanhu/.rbenv/versions/2.1.6/lib/ruby/gems/2.1.0/gems/rouge-1.9.1/lib/rouge/lexers/shell.rb:20: warning: previous definition of KEYWORDS was here
/Users/stanhu/.rbenv/versions/2.1.6/lib/ruby/gems/2.1.0/gems/rouge-1.9.1/lib/rouge/lexers/shell.rb:25: warning: already initialized constant Rouge::Lexers::Shell::BUILTINS
/Users/stanhu/.rbenv/versions/2.1.6/lib/ruby/gems/2.1.0/gems/rouge-1.9.1/lib/rouge/lexers/shell.rb:25: warning: previous definition of BUILTINS was here
```

These loads happen twice:

1. https://github.com/jneen/rouge/blob/master/lib/rouge/lexers/powershell.rb#L5 
1. https://github.com/jneen/rouge/blob/master/lib/rouge.rb#L43

Removing these lines seems to make the warnings go away. Is it possible to remove these automatic loads?

### PR DESCRIPTION
**Pull request by stanhu**
This builds upon https://github.com/rumpelsepp/rugments/commit/e6bdd4eeed47670710d48f53c4b7759a35f1f348, but ensues the specs and `rackup` app work.

Closes #296

### CODE REVIEW

**Comment by rumpelsepp** [comment]
There are more `load` things I think: #291

**Comment by stanhu** [comment]
@rumpelsepp Thanks, I updated the pull request. There are also more `load_const` calls as well. I'm not sure if #291 works with `rackup`, though. Manually cleaning up $LOADED_FEATUES was required.

**Comment by rodrigoargumedo** [comment]
This will also need to close #290. This seems to be good. Seeking a merge on this one.

**Comment by stanhu** [comment]
@jneen, any thoughts?

File: lib/rouge.rb [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 # stdlib
 require 'pathname'
+require_relative 'rouge/version'
```
Comment (segiddins) [inline lib/rouge.rb]: `require_relative` isn't available on 1.8.7

Comment (stanhu) [inline lib/rouge.rb]: Given the [.travis.yml](https://github.com/jneen/rouge/blob/master/.travis.yml) seems to test a minimum version of Ruby v1.9.3, I don't think this should be a problem.

**Comment by esmevane** [comment]
:+1:

**Comment by mhutter** [comment]
:+1:

**Comment by jacobvosmaer** [comment]
:+1:

File: lib/rouge.rb (hunk 1 above)
Comment (JoshCheek) [inline lib/rouge.rb]: But why not just use require?

Comment (dentarg) [inline lib/rouge.rb]: Maybe the intent is more clear with require_relative

Comment (rodrigoargumedo) [inline lib/rouge.rb]: I think `require_relative` is okay, for now. @jneen what are your thoughts on this?

**Comment by mkarbowiak** [comment]
:+1:

**Comment by MrMarvin** [comment]
Are there any issues with this left open? I'd like to get this merged to get the warnings in my application disappear. Look good to me :+1:

**Comment by vincentwoo** [comment]
:+1:

**Comment by wolf99** [comment]
:+1:

**Comment by n1moe1n** [comment]
+1

**Comment by wvolz** [comment]
:+1:

**Comment by jneen** [comment]
Closing for now, and seconding the intent to not depend on `require_relative`.

**Comment by jneen** [comment]
This should be fixed in v1.10.0.

**Comment by stanhu** [comment]
Thanks and welcome back, @jneen! Glad to see the new release.

I already submitted an update to GitLab:

https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/1277

**Comment by jneen** [comment]
:]

**Pull request closed without merging**